testing/peervpn/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Markus Juenemann <markus@juenemann.net>
+# Maintainer: Markus Juenemann <markus@juenemann.net>
+pkgname=peervpn
+pkgver=0.044
+pkgrel=0
+pkgdesc="PeerVPN builds virtual ethernet networks between multiple computers"
+url="https://peervpn.net"
+arch="all"
+license="GPL3"
+depends="openssl zlib"
+makedepends="$depends openssl-dev zlib-dev linux-headers"
```

Comment:
We’ve migrated to LibreSSL in Alpine v3.5. Are you preparing new aports on edge or at least v3.5?